app/scripts/lib/multichain-api/provider-authorize.js

```diff
@@ -93,7 +74,10 @@ export const flattenScopes = (scopes) => {
   return flattenedScopes;
 };
 
-export const assertScopesSupported = (scopes, findNetworkClientIdByChainId) => {
+export const assertScopesSupported = (
+  scopes,
+  { findNetworkClientIdByChainId, getInternalAccounts },
+) => {
   // TODO: Should we be less strict validating optional scopes? As in we can
   // drop parts or the entire optional scope when we hit something invalid which
   // is not true for the required scopes.
```

Comment:
this is a good question 🤔 for now lets be strict on this but we may want to change that later